FirebaseVertexAI/CHANGELOG.md

```diff
@@ -5,8 +5,10 @@
 - [changed] **Breaking Change**: The `BlockThreshold` enum in `SafetySetting`
   has been renamed to `HarmBlockThreshold`. (#13696)
 - [changed] **Breaking Change**: The `unspecified` case has been removed from
-  the `FinishReason`, `PromptFeedback` and `HarmProbability` enums; this
-  scenario is now handled by the existing `unknown` case. (#13699)
+  the `FinishReason`, `BlockReason` and `HarmProbability` enums; this scenario
```

Comment:
This is just a typo fix from before (`PromptFeedback` should have been `BlockReason`).